pkgs/tools/misc/dasht/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, coreutils
+, sed
+, sqlite
+, wget
+, w3m
+, socat
+, gawk
+}:
+
+stdenv.mkDerivation rec {
+  version = "2.3.0";
+  name    = "dasht-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "sunaku";
+    repo   = "dasht";
+    rev    = "v${version}";
+    sha256 = "0d0pcjalba58nvxdgn39m4b6n9ifajf3ygyjaqgvzwxzgpzw0a60";
+  };
+
+  deps = lib.makeBinPath [ coreutils sed sqlite wget w3m socat gawk (placeholder "out") ];
+
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp bin/* $out/bin/
+
+    mkdir -p $out/share/man
+    cp man/man1/* $out/share/man/
+
+    for i in $out/bin/*; do
+      echo "Wrapping $i"
+      wrapProgram $i --prefix PATH : ${deps};
+    done;
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Search API docs offline, in terminal or browser";
+    homepage    = https://sunaku.github.io/dasht/man;
+    license     = stdenv.lib.licenses.isc;
+    platforms   = stdenv.lib.platforms.linux; #cannot test other
```

I think we can assume unix here
```suggestion
    platforms   = stdenv.lib.platforms.unix; #cannot test other
```